app/javascript/components/maps/components/menu/components/sections/explore/explore-component.jsx

```diff
@@ -39,14 +39,6 @@ class Explore extends PureComponent {
           setMenuSettings({ exploreType: 'placesToWatch' });
           track('mapMenuExploreCategory', { label: 'Places to watch' });
         }
-      },
-      {
-        label: 'Stories',
```

Comment:
We need this. We aren't deleting the stories section, only the user stories. Check the stories config to remove it there.